pkg/util/annotation_test.go

```diff
@@ -361,6 +361,47 @@ func TestRecordManagedAnnotations(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "object has has annotations and labels",
```

Comment:
It has the same case as the above case, how can we distinguish them?

Reply:
Sure I will update with suitable case, also in the other name "has" is 2 times I will rectify that also...